src/library/scala/collection/immutable/NumericRange.scala

```diff
@@ -383,15 +440,19 @@ object NumericRange {
           if (step == num.fromInt(stepint)) {
             return {
               if (isInclusive) Range.inclusive(startint, endint, stepint).length
-              else             Range          (startint, endint, stepint).length
+              else Range(startint, endint, stepint).length
```

Comment:
Whitespace was parallel here in order to make it instantly visually obvious whether the arguments are the same (as they should be)

Reply:
That's a good one. I recently started browsing with intellij, where it looks like
![image](https://github.com/scala/scala/assets/369425/443c75f5-f6db-4a67-87b1-e85c24751248)

Reply:
Hahaha, you can't win!